tools/gen_ort_dockerfile.py

```diff
@@ -32,6 +32,13 @@
 
 FLAGS = None
 
+ORT_TO_TRTPARSER_VERSION_MAP = {
+    '1.9.1': (
+        '21.12',         # Triton container version
```

Comment:
Do we need the Triton container version? Can't we just have a map from ORT version to corresponding parser version?

Reply:
No it is not enough... we either need triton version or trt version... because which version of parser to pick depends on the trt version. I chose to use triton container version instead of trt version here because we already have the info regarding triton version so simply using it is easy.

Reply:
The problem is that linux x86, linux arm, jetson and windows may use different versions of TensorRT in each release. I think we need to extract the TRT version from the image in build.py and then pass it to ORT backend build by another ARG. Then your logic would be as it is now except using TRT version (if given) instead of Triton version. I can make the build.py changes to determine the TRT version.

Reply:
ok makes sense. I will update this PR to use trt version instead...